libwhisper/WhisperHost.cpp

```diff
@@ -114,6 +120,8 @@ unsigned WhisperHost::installWatch(shh::Topics const& _t)
 		m_filters.insert(make_pair(h, f));
 
 	m_bloom.addRaw(f.filter.exportBloom());
+	advertizeTopicsOfInterest();
```

Comment:
Would recommend removing this so that advertisements are periodic. I would add that I hope we will change the periodic interval to a random one.

I suspect that doing that performing an immediate advertisement helps with tests. Accordingly, I'd recommend a noteTopicFilterChanged with a "forceAdvertisement" bool which will cause an immediate advertisement of the new topics. We can add this flag to install/uninstall and percolate it through so that changes can be tested without waiting on interval; this also gives us a tool for when we want to make changes to increase or reduce false-positive ratio.